src/main/java/com/synopsys/integration/detect/ApplicationUpdater.java

```diff
@@ -226,6 +231,8 @@ private boolean runMainClass(Path jarPath, String[] launchArgs)
             final Class<?> launcherClass = 	classMap.get("org.springframework.boot.loader.Launcher");
             final Method launchMethod = launcherClass.getDeclaredMethod("launch", String[].class);
             launchMethod.setAccessible(true);
+            checkEnvironmentProperties();
+            args = parseArguments(args);
```

Comment:
These two checks are needed to prevent failure in the airgapped gradle build tasks that rely on System environment properties.